web/ts/watch.ts

```diff
@@ -69,9 +109,15 @@ export function startWebsocket() {
                 window.dispatchEvent(new CustomEvent("pauseend"));
             }
         } else if ("server" in data) {
+            const scroll = shouldScroll();
+            console.log("scroll: ", scroll);
             const serverElem = createServerMessage(data);
             document.getElementById("chatBox").appendChild(serverElem);
-            scrollChatIfNeeded();
+            if (scroll) {
+                setTimeout(scrollChat, 100); // wait a bit to make sure the message is added before scrolling
```

Comment:
Maybe we could extract `100` into a constant and call it `scrollDelay`.

Reply:
:+1: